config/azure.spc

```diff
@@ -1,12 +1,36 @@
 connection "azure" {
```

Comment:
The `azuread` plugin also has config args for using msi, `enable_msi` and `msi_endpoint`, do we need those here as well? If not, do we need them in the `azuread` plugin?